mariner/mariner.go

```diff
@@ -154,8 +152,24 @@ func (c Config) update(version, path string) error {
 
 	return nil
 }
+func (c Config) saveAdvisoryPerYear(dirName string, def Definition) error {
+	// Mariner uses `<ID>_<last_number_from_version>` format for `advisory_id`.
+	// But `advisory_id` is not required field.
+	// Therefore, if `advisory_id` is not exist, we create this field independently.
+	// cf. https://github.com/aquasecurity/vuln-list-update/pull/271#issuecomment-2111678641
+	advisoryID := def.Metadata.AdvisoryID
+	if advisoryID == "" {
+		advisoryID = def.ID
```

Comment:
The ID is something like `oval:com.microsoft.cbl-mariner:obj:31880001`. Shouldn't we extract the last digits, `31880001`?

Reply:
Nice catch!
Changed in https://github.com/aquasecurity/vuln-list-update/pull/271/commits/995ea97f301c04bacd37987086c6d8992add1703
Also added tests - https://github.com/aquasecurity/vuln-list-update/pull/271/commits/7074604717e7a6a93d6a4a689d4098cc9b7268bd